OutOfSchool/OutOfSchool.WebApi/Services/ParentService.cs

```diff
@@ -140,7 +149,7 @@ public async Task<ParentDTO> GetByUserId(string id)
 
         logger.LogInformation($"Successfully got a Parent with UserId = {id}.");
 
-        return parents.FirstOrDefault().ToModel();
+        return mapper.Map<ParentDTO>(parents.FirstOrDefault());
```

Comment:
Looks like a possible NRE in the mapper, or check for `null` is redundant in the `ChatRoomWorkshopervice` ?

Reply:
I saw that but I didn't want to change any logic, just replace extensions with IMapper alternative. For now `mapper.Map<ParentDTO>` will return `null`, not NRE but I also agree that it'd be better to check for nulls like in the `ChatRoomWorkshopervice `.